theme/lumo/vaadin-message-input-styles.js

```diff
@@ -0,0 +1,25 @@
+import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import '@vaadin/vaadin-lumo-styles/color.js';
+import '@vaadin/vaadin-lumo-styles/sizing.js';
+import '@vaadin/vaadin-lumo-styles/spacing.js';
+import '@vaadin/vaadin-lumo-styles/style.js';
```

Comment:
Do we have some convention that should all of these be imported always or only those that you refer to in the same file? Spacing seems to be the only one in use below.